### PR TITLE
[EDU-647] - Fix get device reg info

### PR DESCRIPTION
**Pull request by tbedford**
## Description

Fix the get device registration JSON block, and add info on `push.error`.

* [EDU-647](https://ably.atlassian.net/browse/EDU-647)

## Review

* [REST AI - get device reg](https://ably-docs-edu-647-error-lo5dqw.herokuapp.com/api/rest-api/#get-device-registration)

### CODE REVIEW

File: content/api/rest-api.textile [hunk 1]
```diff
@@ -506,11 +506,18 @@ bc[json]. {
...
+      statusCode: <string>,
+      message: <string>
+    }
   }
 }
 
-or a 404 error if a device by that ID does not exist.
+or a 404 error if a device by that ID does not exist. 
+
+If Ably could not register the device on a behalf of the push notifications provider, then @push.error@ contains the relevant error information.
```
Comment (Sapples) [inline content/api/rest-api.textile:520]: "If Ably could not register the device on **~~a~~**  behalf of the push notifications provider, then @push.error@ contains the relevant error information."

Extraneous 'a'.

**Review by Sapples — CHANGES_REQUESTED** [review]
One tiny typo.

File: content/api/rest-api.textile (hunk 1 above)
Comment (tbedford) [reply content/api/rest-api.textile]: well spotted! :)

**Review by Sapples — APPROVED** [review]
LGTM!

**Pull request merged**